services/object/src/backends/aws.js

```diff
@@ -18,8 +18,19 @@ class AwsBackend extends Backend {
       accessKeyId: this.config.accessKeyId,
       secretAccessKey: this.config.secretAccessKey,
     };
+
+    try{
     this.region = await getBucketRegion({ bucket: this.config.bucket, credentials });
+    }catch(err){
+      return;
+    }
+    
+    try{
     this.s3 = new aws.S3({ region: this.region, ...credentials });
+    }catch(err){
+      return;
```

Comment:
The effect of this change is just to ignore erorrs!  We definitely don't want to do that.  This issue is about reporting errors and allowing other providers to continue to operate, with just the failed provider not working.

Reply:
I am a bit confused. so I have to  add an error message or remove try-catch

Reply:
Hm, I got a notification of a comment here, but now I don't see it.. did it get deleted?  Deleting comments in a conversation is really confusing!

Reply:
ok,
sorry for that 
I have made some changes